To-do list/Controllers/Cell/TaskTableViewCell.swift

```diff
@@ -20,16 +20,15 @@ class TaskTableViewCell: UITableViewCell {
     @IBOutlet weak var deadlineLabel: UILabel!
     @IBOutlet weak var statusButtonOutlet: UIButton!
     
- 
-    override func awakeFromNib() {
-        super.awakeFromNib()
-    }
 
-    override func setSelected(_ selected: Bool, animated: Bool) {
-        super.setSelected(selected, animated: animated)
-    }
-    
     @IBAction func statusButton(_ sender: UIButton) {
         delegate?.didTapStatusButton(cell: self)
```

Comment:
This cell should keep reference to Task model passed in `func configure(_ task: Task)`. Then, when status button is clicked, delegate func should pass it `delegate?.taskTableView(cell: self, didClickOnTaskButton task: Task)`

Reply:
please more details

Reply:
done